src/safeds/ml/nn/_model.py

```diff
@@ -0,0 +1,186 @@
+import numpy as np
+import torch
+from torch import nn
+
+from safeds.data.tabular.containers import TaggedTable
+from safeds.exceptions import ClosedBound, OutOfBoundsError
+from safeds.ml.nn._fnn_layer import FNNLayer
+
+
+class RegressionModel:
+    def __init__(self, layers: list):
+        self._model = _PytorchModel(layers, is_for_classification=False)
+        self._batch_size = 1
+
+    def train(self, train_data: TaggedTable, epoch_size: int = 25, batch_size: int = 1) -> None:
```

Comment:
Try to be consistent with the other models. There, the function is called fit.

Reply:
The function must return the trained model and not None. The original model must not be mutated.